### PR TITLE
cbr backup modules

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

**Review by vladimirvshivkov — APPROVED** [review]

File: plugins/modules/cbr_restore_point.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+DOCUMENTATION = '''
+module: cbr_restore_point
+short_description: Manage CBR Restore Point
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.12.4"
+author: "Gubina Polina (@Polina-Gubina)"
+description:
+    - Manage cbr restore point from the OTC.
+options:
+  vault:
```
Comment (anton-sidelnikov) [inline plugins/modules/cbr_restore_point.py:23]: please sort parameters alphabetically or by majority

Comment (anton-sidelnikov) [reply plugins/modules/cbr_restore_point.py:23]: in all modules

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: plugins/modules/cbr_restore_point.py (hunk 1 above)
Comment (Polina-Gubina) [reply plugins/modules/cbr_restore_point.py:23]: all parametrs are sorted alphabetically but vault is first because it is madatory paramete
r

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by vladimirvshivkov** [comment]
regate

**Pull request merged**